Can't compile debug template on latest master

### Godot version

v4.0.beta.custom_build [43e9ca40f]

### System information

Manjaro Linux

### Issue description

Fails to compile with:

`scene/debugger/scene_debugger.cpp: In static member function 'static void SceneDebugger::_save_node(ObjectID, const String&)':
scene/debugger/scene_debugger.cpp:224:28: error: 'class Node' has no member named 'duplicate_from_editor'
  224 |         Node *copy = node->duplicate_from_editor(duplimap);
      |                            ^~~~~~~~~~~~~~~~~~~~~
`

### Steps to reproduce

just compile with  `scons target=template_debug arch=x86_64 -j "$(nproc)"`

### Minimal reproduction project

Not needed

--- a/scene/debugger/scene_debugger.cpp
+++ b/scene/debugger/scene_debugger.cpp
@@ -220,8 +220,12 @@ void SceneDebugger::_save_node(ObjectID id, const String &p_path) {
 	Node *node = Object::cast_to<Node>(ObjectDB::get_instance(id));
 	ERR_FAIL_COND(!node);
 
+#ifdef TOOLS_ENABLED
 	HashMap<const Node *, Node *> duplimap;
 	Node *copy = node->duplicate_from_editor(duplimap);
+#else
+	Node *copy = node->duplicate();
+#endif
 
 	// Handle Unique Nodes.
 	for (int i = 0; i < copy->get_child_count(false); i++) {

